core/src/main/java/org/hisp/dhis/android/core/trackedentity/TrackedEntityInstance.java

```diff
@@ -67,6 +69,8 @@
     public static final Field<TrackedEntityInstance, String> trackedEntity = Field.create(
             TRACKED_ENTITY);
     public static final Field<TrackedEntityInstance, Boolean> deleted = Field.create(DELETED);
+    public static final Field<TrackedEntityInstance, State> state = Field.create(
```

Comment:
This field is not being used, is it necessary?

Reply:
I added it because this field is persisted in db but never recover, but  when i try to push a tei with state and i needed comment it to make it work again.
I am not sure if remove the field is a good solution. But i will do it the next commit.